doc/source/whatsnew/v1.1.0.rst

```diff
@@ -298,6 +298,7 @@ Other enhancements
 - :meth:`~pandas.io.gbq.read_gbq` now allows to disable progress bar (:issue:`33360`).
 - :meth:`~pandas.io.gbq.read_gbq` now supports the ``max_results`` kwarg from ``pandas-gbq`` (:issue:`34639`).
 - :meth:`DataFrame.to_html` and :meth:`DataFrame.to_string`'s ``col_space`` parameter now accepts a list of dict to change only some specific columns' width (:issue:`28917`).
+- :meth:`~Series.explode` now accepts ``ignore_index`` to reset the index, similarly to ``pd.concat`` or ``DataFrame.sort_values`` (:issue:`34932`).
```

Comment:
can you add these as `:meth:` refs as well

Reply:
I'm sorry I can't figure out what you mean with this, would you mind helping me a bit here?

Reply:
```suggestion
- :meth:`~Series.explode` now accepts ``ignore_index`` to reset the index, similarly to :meth:`pd.concat` or :meth:`DataFrame.sort_values` (:issue:`34932`).
```